src/serac/physics/state/state_manager.cpp

```diff
@@ -221,6 +233,26 @@ void StateManager::save(const double t, const int cycle, const std::string& mesh
 
 mfem::ParMesh* StateManager::setMesh(std::unique_ptr<mfem::ParMesh> pmesh, const std::string& mesh_tag)
 {
+  // Determine if the existing nodal grid function is discontinuous. This
+  // indicates that the mesh is periodic and the new nodal grid function must also
+  // be discontinuous.
+  bool is_discontinuous = false;
+  auto nodes            = pmesh->GetNodes();
```

Comment:
Can't `GetNodes()` return `NULL` for other reasons (unrelated to periodicity of the mesh)?

I've asked mfem to clarify the preconditions of this function before, but it's still essentially undocumented.

Reply:
Ah, I misunderstood what this was doing